colossalai/inference/tensor_parallel/batch_infer_state.py

```diff
@@ -5,7 +5,7 @@
 
 from .kvcache_manager import MemoryManager
 
-
+# adaped from: lightllm/server/router/model_infer/infer_batch.py
```

Comment:
Adaped should be adapted. Follows are same error.

Reply:
fixed .